lib/modules/manager/flux/common.ts

```diff
@@ -1,8 +1,11 @@
 import { regEx } from '../../../util/regex';
 
-export const systemManifestRegex =
-  '(^|/)flux-system/(?:.+/)?gotk-components\\.ya?ml$';
+export const systemManifestFileNameRegex =
+  '^(?:.+/)?gotk-components\\.ya?ml$';
```

Comment:
```suggestion
export const systemManifestFileNameRegex =
  '(?:^|/)gotk-components\\.ya?ml$';
```
this prefix is much simpler and probably faster 😉

Reply:
Ooh great spot!